opensrp-anc/src/main/java/org/smartregister/anc/library/activity/ContactJsonFormActivity.java

```diff
@@ -164,7 +164,7 @@ public void onResume() {
             formName = getIntent().getStringExtra(ConstantsUtils.IntentKeyUtils.FORM_NAME);
         }
         try {
-            ANCFormUtils.processCheckboxFilteredItems(mJSONObject);
+            org.smartregister.anc.library.util.ANCFormUtils.processCheckboxFilteredItems(mJSONObject);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.anc.library.util.ANCFormUtils.processCheckboxFilteredItems' due to existing import 'org.smartregister.anc.library.util.ANCFormUtils'](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=5257516)